clients/go/apiban-ipsets/config.ini

```diff
@@ -0,0 +1,21 @@
+# apiban-ipsets configuration file
+# SSO token
+TOKEN="authtokenstring"
+# lk id
+LKID="0"
+# URL of white/blacklisted entries in DB
+URL="https://siem.intuitivelabs.com/api/"
+# iptables chain name for matching entries
+CHAIN="BLOCKER"
+# interval for the list refresh
+INTERVAL="60s"
+# yes/no - starting from scratch
+FULL="yes"
+# default blacklisted entry timeout in seconds
+BLACKLIST_TTL="0s"
+# filename for keeping the state
+STATE_FILENAME="/tmp/apiban.state"
+# password for encryption
+PASSPHRASE="foobar"
```

Comment:
I did not test any `PASSPHRASE` use-cases with api client since we do not have any yet. only tested with `ENCRYPTION_KEY`

Reply:
understand, but this is the config file as part of the RPM. We update it when using the installer script that our subscriber gets from GUI, and only the encryption_key is used there. So in theory, our users will not use passphrase. :-) But I can test...